src/platforms/react-native/troubleshooting.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Troubleshooting
+description: "Troubleshoot and resolve common issues with the React Native SDK."
+sidebar_order: 1000
+---
+
+## Source Maps
+
+When there is an issue with source maps on your event, there will usually be an error dialog at the top of the event page on your dashboard:
+
+> There was 1 problem processing this event: Source code was not found
+
+Common fixes to this are:
+
+- Make sure that both the `release` and `dist` and are set on the event by [setting it in the call to `init`](/platforms/react-native/configuration/releases/#bind-the-version).
+- Make sure that the source maps are correctly uploaded with the exact same `release` and `dist` values as the event.
+- If you set a custom `release` and `dist` make sure that you manually upload the sourcemaps as our automatic sourcemap upload script will not detect the custom values.
+- Make sure that you are not disabling the `RewriteFrames` default integration by filtering it out or setting `defaultIntegrations: false`.
```

Comment:
```suggestion
- Checking that you are not disabling the `RewriteFrames` default integration by filtering it out or setting `defaultIntegrations: false`.
```

Reply:
We don't seem to have `RewriteFrames` (or any integrations) documented for RN. Do we want to link to the content in JS?

Reply:
Yes we should!

Reply:
@PeloWriter Actually, additional thought is that we should include the integrations content in RN as well. All the pluggable integrations and most (but not all) of the default integrations are in RN as well.

Reply:
That makes sense. The sad part is that we don't have the ability (yet) to inherit from siblings (that is, since RN is at the same level as the JS content, we can't just inherit the JS content). The best way (it's a kludge for sure) to resolve this is to build those pages out, then `include` them in both RN and JS. Alternately, just linking works for now until we resolve this issue.